parcels/kernel.py

```diff
@@ -178,6 +179,8 @@ def __init__(self, fieldset, ptype, pyfunc=None, funcname=None, funccode=None, p
         else:
             self.funcvars = None
         self.funccode = funccode or inspect.getsource(pyfunc.__code__)
+        self.funccode = self.funccode.replace('parcels.rng', 'rng')  # Remove parcels. prefix (see #1608)
```

Comment:
If this is the entry point, would it be easier to just replace this with 'ParcelsRandom'? It would result in less changes to the codebase and everything would just flow through as if its `ParcelsRandom` and also wouldn't interfere in the odd chance they already have a variable called `rng`. So maybe its just this one-liner?

Reply:
True, but I actually like to start using `parcels.rng` a bit more throughout the codebase, as `ParcelsRandom` was just an alias around it. Hence the other changes

Reply:
I see. Sounds good!